packages/perseus/src/interactive2/movable-point.tsx

```diff
@@ -801,5 +801,3 @@ function getKey(eventName: string, id: string): string {
 function getEventName(key: string): string {
     return key.split(":")[0];
 }
-
-export default MovablePoint;
```

Comment:
We really should have a final call on whether we use default exports or named.  I find a single named export kinda weird, but it protects against an import changing the name and being confusing. 🤔